firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -270,15 +286,16 @@ private String createThrottledMessage(long throttledDurationInMillis) {
    *     error connecting to the server.
    */
   @WorkerThread
-  private FetchResponse fetchFromBackend(Date currentTime) throws FirebaseRemoteConfigException {
+  private FetchResponse fetchFromBackend(InstanceIdResult instanceId, Date currentTime)
+      throws FirebaseRemoteConfigException {
     try {
       HttpURLConnection urlConnection = frcBackendApiClient.createHttpURLConnection();
 
       FetchResponse response =
           frcBackendApiClient.fetch(
               urlConnection,
-              firebaseInstanceId.getId(),
-              firebaseInstanceId.getToken(),
```

Comment:
Thinking out loud: this continues to be an instance var. Weird how it has `getInstanceId` and `getId` methods.

Reply:
I was just noting the similarity in naming between `firebaseInstanceId.getInstanceId` and `instanceId.getId`. I guess this is highlighted by this review, where we can see both side by side.  To be clear, I don't think any action is required.